web/src/main/java/com/zebrunner/reporting/web/ApiExceptionHandler.java

```diff
@@ -92,22 +93,39 @@ public ErrorResponse handleMethodArgumentNotValidException(Exception e) {
             List<FieldError> fieldErrors = bindingResult.getFieldErrors();
 
             for (FieldError fieldError : fieldErrors) {
-                Error error = new Error(ErrorCode.INVALID_VALUE, fieldError.getField(), fieldError.getDefaultMessage());
+                Error error = buildError(fieldError.getField(), fieldError.getDefaultMessage(), fieldError.getRejectedValue());
+                response.getValidationErrors().add(error);
+            }
+        }
+
+        return response;
+    }
+
+    @ExceptionHandler(ConstraintViolationException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public ErrorResponse handleConstraintViolationException(ConstraintViolationException e) {
```

Comment:
To my mind, explicit handling of `ConstraintViolationException` is a design smell.